templates/vue/config-sample.js

```diff
@@ -1 +0,0 @@
-export const LINK_PREVIEW_API_KEY = ""
```

Comment:
Please keep this file. We need it for new users so they know what the `config.js` file should look like. It's also unrelated to the deprecated problem (to my understanding). We want to keep the PRs as contained to their original issue as possible.